lib/components/Typography/base.tsx

```diff
@@ -9,13 +9,15 @@ export interface TypographyProps extends ComponentProps<THeaders> {
     bold?: boolean;
     italic?: boolean;
     underlined?: boolean;
+    color?: string;
 }
 
 type THeaders = "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
 
 export const Typography = ({
     children,
     className,
+    color,
```

Comment:
Color here is a `className`?

Reply:
I think @meenakshi-deriv trying to replace the default color in `Typography` component. I guess if we change the color in the corresponding component like H1, H2, code, caption... instead of putting the color classname in the base component.